proto/permissions.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
```

Comment:
is there a diff in this file that we should be paying attention to, or is this just an unchanged migration to a separate file

Reply:
No diff, unchanged migration to a separate file